protocols/kad/src/behaviour.rs

```diff
@@ -802,6 +909,14 @@ pub enum KademliaOut {
         /// List of peers ordered from closest to furthest away.
         closer_peers: Vec<PeerId>,
     },
+
+    /// Result of a `GET_VALUE` query
+    GetValueRes {
```

Comment:
```suggestion
    GetValueResult {
```
Just for naming consistency.